From bf75465e769f417ef3c4acc56d9970fad650e906 Mon Sep 17 00:00:00 2001
From: Nguyen Thanh Nhan 2 <nhan.nguyen.uf@bp.renesas.com>
Date: Tue, 5 Aug 2025 17:24:17 +0700
Subject: [PATCH] rz-common: recipes-debian: buster: switch some packages to
 ELTS mirror

Switch to ELTS mirror to avoid being removed in Debian Mirror as following:
- attr
- coreutils
- gcc-8
- kbd
- libassuan
- libgcrypt20
- libunistring
- nss
- perl
- pkg-config
- pseudo
- quilt
- gnupg2
- gzip

Also update some packages to newer versions as following:
- binutils 2.31.1-16 -> 2.31.1-16+deb10u1
- busybox 1.30.1-4 -> 1.30.1-4+deb10u1
- openssh 7.9p1-10+deb10u2 -> 7.9p1-10+deb10u5
- openssl 1.1.1n-0+deb10u6 -> 1.1.1n-0+deb10u7

Signed-off-by: Nguyen Thanh Nhan 2 <nhan.nguyen.uf@bp.renesas.com>

diff --git a/meta-rz-common/recipes-debian/buster/sources/attr.inc b/meta-rz-common/recipes-debian/buster/sources/attr.inc
index 80afb82b..8cf0c503 100644
--- a/meta-rz-common/recipes-debian/buster/sources/attr.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/attr.inc
@@ -5,10 +5,10 @@ REPACK_PV = "2.4.48"
 PV = "2.4.48"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/a/attr/attr_2.4.48-4.dsc;name=attr_2.4.48-4.dsc \
-    ${DEBIAN_MIRROR}/main/a/attr/attr_2.4.48.orig.tar.gz;name=attr_2.4.48.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/a/attr/attr_2.4.48.orig.tar.gz.asc;name=attr_2.4.48.orig.tar.gz.asc \
-    ${DEBIAN_MIRROR}/main/a/attr/attr_2.4.48-4.debian.tar.xz;name=attr_2.4.48-4.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/a/attr/attr_2.4.48-4.dsc;name=attr_2.4.48-4.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/a/attr/attr_2.4.48.orig.tar.gz;name=attr_2.4.48.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/a/attr/attr_2.4.48.orig.tar.gz.asc;name=attr_2.4.48.orig.tar.gz.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/a/attr/attr_2.4.48-4.debian.tar.xz;name=attr_2.4.48-4.debian.tar.xz \
 "
 
 SRC_URI[attr_2.4.48-4.dsc.md5sum] = "6572c0f8d4a184962e026faf399afded"
diff --git a/meta-rz-common/recipes-debian/buster/sources/binutils.inc b/meta-rz-common/recipes-debian/buster/sources/binutils.inc
index 40e60ba5..fbba384e 100644
--- a/meta-rz-common/recipes-debian/buster/sources/binutils.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/binutils.inc
@@ -1,19 +1,19 @@
 # This is generated by debian-source.bbclass
-DPV = "2.31.1-16"
+DPV = "2.31.1-16+deb10u1"
 DPV_EPOCH = ""
 REPACK_PV = "2.31.1"
 PV = "2.31.1"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/b/binutils/binutils_2.31.1-16.dsc;name=binutils_2.31.1-16.dsc \
-    ${DEBIAN_MIRROR}/main/b/binutils/binutils_2.31.1.orig.tar.xz;name=binutils_2.31.1.orig.tar.xz \
-    ${DEBIAN_MIRROR}/main/b/binutils/binutils_2.31.1-16.debian.tar.xz;name=binutils_2.31.1-16.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/b/binutils/binutils_2.31.1-16+deb10u1.dsc;name=binutils_2.31.1-16+deb10u1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/b/binutils/binutils_2.31.1.orig.tar.xz;name=binutils_2.31.1.orig.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/b/binutils/binutils_2.31.1-16+deb10u1.debian.tar.xz;name=binutils_2.31.1-16+deb10u1.debian.tar.xz \
 "
 
-SRC_URI[binutils_2.31.1-16.dsc.md5sum] = "5f074f055b95a97bc3f01773e050d4ff"
+SRC_URI[binutils_2.31.1-16+deb10u1.dsc.md5sum] = "e9cb5b09cab1070d958939c98eeafc5a"
 SRC_URI[binutils_2.31.1.orig.tar.xz.md5sum] = "5f07a94a7eee4c1e3fce85a9225f3bd3"
-SRC_URI[binutils_2.31.1-16.debian.tar.xz.md5sum] = "0655a52c4a1a0d26e0ca85fffbbf9b6f"
+SRC_URI[binutils_2.31.1-16+deb10u1.debian.tar.xz.md5sum] = "3c94cc53a4f394c1484dd2393530c084"
 
-SRC_URI[binutils_2.31.1-16.dsc.sha256sum] = "ec76c13684d922a3619d7ec982db191714927bde6de6a3ff89e95d1ce7a61f33"
+SRC_URI[binutils_2.31.1-16+deb10u1.dsc.sha256sum] = "fa418ec299d719961227bb173c9522ba4854ed29ee1ac9aadd563816276c6615"
 SRC_URI[binutils_2.31.1.orig.tar.xz.sha256sum] = "e398a2d579faa0f2b5a988add5f7481af8e21a21f63b6ea5702e6f517960c5eb"
-SRC_URI[binutils_2.31.1-16.debian.tar.xz.sha256sum] = "15fc82a7c682da6bcbf56caf57da8f059655369cbfeb58b8312040e53e4fa11d"
+SRC_URI[binutils_2.31.1-16+deb10u1.debian.tar.xz.sha256sum] = "f16bf0b487bccb39755eb7db80680f74ed9135ba31739d96c73e2299917aa65b"
diff --git a/meta-rz-common/recipes-debian/buster/sources/busybox.inc b/meta-rz-common/recipes-debian/buster/sources/busybox.inc
index 9c79d127..0efc4f55 100644
--- a/meta-rz-common/recipes-debian/buster/sources/busybox.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/busybox.inc
@@ -1,19 +1,19 @@
 # This is generated by debian-source.bbclass
-DPV = "1.30.1-4"
+DPV = "1.30.1-4+deb10u1"
 DPV_EPOCH = "1"
 REPACK_PV = "1.30.1"
 PV = "1.30.1"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/b/busybox/busybox_1.30.1-4.dsc;name=busybox_1.30.1-4.dsc \
-    ${DEBIAN_MIRROR}/main/b/busybox/busybox_1.30.1.orig.tar.bz2;name=busybox_1.30.1.orig.tar.bz2 \
-    ${DEBIAN_MIRROR}/main/b/busybox/busybox_1.30.1-4.debian.tar.xz;name=busybox_1.30.1-4.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/b/busybox/busybox_1.30.1-4+deb10u1.dsc;name=busybox_1.30.1-4+deb10u1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/b/busybox/busybox_1.30.1.orig.tar.bz2;name=busybox_1.30.1.orig.tar.bz2 \
+    ${DEBIAN_ELTS_MIRROR}/main/b/busybox/busybox_1.30.1-4+deb10u1.debian.tar.xz;name=busybox_1.30.1-4+deb10u1.debian.tar.xz \
 "
 
-SRC_URI[busybox_1.30.1-4.dsc.md5sum] = "2a32c3f30073f7cd529901368c9f7240"
+SRC_URI[busybox_1.30.1-4+deb10u1.dsc.md5sum] = "a31cf5d5987cdda1a6857338f6c04c15"
 SRC_URI[busybox_1.30.1.orig.tar.bz2.md5sum] = "4f72fc6abd736d5f4741fc4a2485547a"
-SRC_URI[busybox_1.30.1-4.debian.tar.xz.md5sum] = "82d8423805f7e1c21dfb64678dd30873"
+SRC_URI[busybox_1.30.1-4+deb10u1.debian.tar.xz.md5sum] = "0dbf1d4c7ae2dbab0d5be5073040ba7e"
 
-SRC_URI[busybox_1.30.1-4.dsc.sha256sum] = "ff1689dbb37c49f8216e83de2368196f72fb63afcf1695487e11a58cab7d91f0"
+SRC_URI[busybox_1.30.1-4+deb10u1.dsc.sha256sum] = "e3eec2dab487ada8d1cf20709d2d3908ce933bfd37f3b4e52fe1cf0f880e6794"
 SRC_URI[busybox_1.30.1.orig.tar.bz2.sha256sum] = "3d1d04a4dbd34048f4794815a5c48ebb9eb53c5277e09ffffc060323b95dfbdc"
-SRC_URI[busybox_1.30.1-4.debian.tar.xz.sha256sum] = "22799c84101c9b85a3afdb82c3b26098fc40db84211738725e8e1ace5d809ae7"
+SRC_URI[busybox_1.30.1-4+deb10u1.debian.tar.xz.sha256sum] = "1ed2df1855bd158013465bc1c6d5fb4b8ccbef97251e4fcd34a85c8cb0e5d2fd"
diff --git a/meta-rz-common/recipes-debian/buster/sources/coreutils.inc b/meta-rz-common/recipes-debian/buster/sources/coreutils.inc
index aec48494..204e3fef 100644
--- a/meta-rz-common/recipes-debian/buster/sources/coreutils.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/coreutils.inc
@@ -5,9 +5,9 @@ REPACK_PV = "8.30"
 PV = "8.30"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/c/coreutils/coreutils_8.30-3.dsc;name=coreutils_8.30-3.dsc \
-    ${DEBIAN_MIRROR}/main/c/coreutils/coreutils_8.30.orig.tar.xz;name=coreutils_8.30.orig.tar.xz \
-    ${DEBIAN_MIRROR}/main/c/coreutils/coreutils_8.30-3.debian.tar.xz;name=coreutils_8.30-3.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/c/coreutils/coreutils_8.30-3.dsc;name=coreutils_8.30-3.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/c/coreutils/coreutils_8.30.orig.tar.xz;name=coreutils_8.30.orig.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/c/coreutils/coreutils_8.30-3.debian.tar.xz;name=coreutils_8.30-3.debian.tar.xz \
 "
 
 SRC_URI[coreutils_8.30-3.dsc.md5sum] = "8e7167ff50149c83192535df18eee396"
diff --git a/meta-rz-common/recipes-debian/buster/sources/gcc-8.inc b/meta-rz-common/recipes-debian/buster/sources/gcc-8.inc
index 640a686a..febc5517 100644
--- a/meta-rz-common/recipes-debian/buster/sources/gcc-8.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/gcc-8.inc
@@ -5,9 +5,9 @@ REPACK_PV = "8.3.0"
 PV = "8.3.0"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/g/gcc-8/gcc-8_8.3.0-6.dsc;name=gcc-8_8.3.0-6.dsc \
-    ${DEBIAN_MIRROR}/main/g/gcc-8/gcc-8_8.3.0.orig.tar.gz;name=gcc-8_8.3.0.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/g/gcc-8/gcc-8_8.3.0-6.diff.gz;name=gcc-8_8.3.0-6.diff.gz;apply=no \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gcc-8/gcc-8_8.3.0-6.dsc;name=gcc-8_8.3.0-6.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gcc-8/gcc-8_8.3.0.orig.tar.gz;name=gcc-8_8.3.0.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gcc-8/gcc-8_8.3.0-6.diff.gz;name=gcc-8_8.3.0-6.diff.gz;apply=no \
 "
 
 SRC_URI[gcc-8_8.3.0-6.dsc.md5sum] = "15e19a0b8b3dbe56437618f78edcc0b3"
diff --git a/meta-rz-common/recipes-debian/buster/sources/gnupg2.inc b/meta-rz-common/recipes-debian/buster/sources/gnupg2.inc
index 38ea58e1..78e0285b 100644
--- a/meta-rz-common/recipes-debian/buster/sources/gnupg2.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/gnupg2.inc
@@ -5,10 +5,10 @@ REPACK_PV = "2.2.12"
 PV = "2.2.12"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gnupg2/gnupg2_2.2.12-1+deb10u2.dsc;name=gnupg2_2.2.12-1+deb10u2.dsc \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gnupg2/gnupg2_2.2.12.orig.tar.bz2;name=gnupg2_2.2.12.orig.tar.bz2 \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gnupg2/gnupg2_2.2.12.orig.tar.bz2.asc;name=gnupg2_2.2.12.orig.tar.bz2.asc \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gnupg2/gnupg2_2.2.12-1+deb10u2.debian.tar.xz;name=gnupg2_2.2.12-1+deb10u2.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gnupg2/gnupg2_2.2.12-1+deb10u2.dsc;name=gnupg2_2.2.12-1+deb10u2.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gnupg2/gnupg2_2.2.12.orig.tar.bz2;name=gnupg2_2.2.12.orig.tar.bz2 \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gnupg2/gnupg2_2.2.12.orig.tar.bz2.asc;name=gnupg2_2.2.12.orig.tar.bz2.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gnupg2/gnupg2_2.2.12-1+deb10u2.debian.tar.xz;name=gnupg2_2.2.12-1+deb10u2.debian.tar.xz \
 "
 
 SRC_URI[gnupg2_2.2.12-1+deb10u2.dsc.md5sum] = "5585917b8d5905559eb08a83fe5caa49"
diff --git a/meta-rz-common/recipes-debian/buster/sources/gzip.inc b/meta-rz-common/recipes-debian/buster/sources/gzip.inc
index eb546d79..ea2c5702 100644
--- a/meta-rz-common/recipes-debian/buster/sources/gzip.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/gzip.inc
@@ -5,9 +5,9 @@ REPACK_PV = "1.9"
 PV = "1.9"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gzip/gzip_1.9-3+deb10u1.dsc;name=gzip_1.9-3+deb10u1.dsc \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gzip/gzip_1.9.orig.tar.gz;name=gzip_1.9.orig.tar.gz \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/g/gzip/gzip_1.9-3+deb10u1.debian.tar.xz;name=gzip_1.9-3+deb10u1.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gzip/gzip_1.9-3+deb10u1.dsc;name=gzip_1.9-3+deb10u1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gzip/gzip_1.9.orig.tar.gz;name=gzip_1.9.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/g/gzip/gzip_1.9-3+deb10u1.debian.tar.xz;name=gzip_1.9-3+deb10u1.debian.tar.xz \
 "
 
 SRC_URI[gzip_1.9-3+deb10u1.dsc.md5sum] = "789b70393726b2e2b4203a61da61d82b"
diff --git a/meta-rz-common/recipes-debian/buster/sources/kbd.inc b/meta-rz-common/recipes-debian/buster/sources/kbd.inc
index abe1bdf3..25b27aa8 100644
--- a/meta-rz-common/recipes-debian/buster/sources/kbd.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/kbd.inc
@@ -5,9 +5,9 @@ REPACK_PV = "2.0.4"
 PV = "2.0.4"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/k/kbd/kbd_2.0.4-4.dsc;name=kbd_2.0.4-4.dsc \
-    ${DEBIAN_MIRROR}/main/k/kbd/kbd_2.0.4.orig.tar.xz;name=kbd_2.0.4.orig.tar.xz \
-    ${DEBIAN_MIRROR}/main/k/kbd/kbd_2.0.4-4.debian.tar.xz;name=kbd_2.0.4-4.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/k/kbd/kbd_2.0.4-4.dsc;name=kbd_2.0.4-4.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/k/kbd/kbd_2.0.4.orig.tar.xz;name=kbd_2.0.4.orig.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/k/kbd/kbd_2.0.4-4.debian.tar.xz;name=kbd_2.0.4-4.debian.tar.xz \
 "
 
 SRC_URI[kbd_2.0.4-4.dsc.md5sum] = "958d849f316af8c8f274cbcf1193e911"
diff --git a/meta-rz-common/recipes-debian/buster/sources/libassuan.inc b/meta-rz-common/recipes-debian/buster/sources/libassuan.inc
index cccd4f79..01f4a866 100644
--- a/meta-rz-common/recipes-debian/buster/sources/libassuan.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/libassuan.inc
@@ -5,10 +5,10 @@ REPACK_PV = "2.5.2"
 PV = "2.5.2"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/liba/libassuan/libassuan_2.5.2-1.dsc;name=libassuan_2.5.2-1.dsc \
-    ${DEBIAN_MIRROR}/main/liba/libassuan/libassuan_2.5.2.orig.tar.bz2;name=libassuan_2.5.2.orig.tar.bz2 \
-    ${DEBIAN_MIRROR}/main/liba/libassuan/libassuan_2.5.2.orig.tar.bz2.asc;name=libassuan_2.5.2.orig.tar.bz2.asc \
-    ${DEBIAN_MIRROR}/main/liba/libassuan/libassuan_2.5.2-1.debian.tar.xz;name=libassuan_2.5.2-1.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/liba/libassuan/libassuan_2.5.2-1.dsc;name=libassuan_2.5.2-1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/liba/libassuan/libassuan_2.5.2.orig.tar.bz2;name=libassuan_2.5.2.orig.tar.bz2 \
+    ${DEBIAN_ELTS_MIRROR}/main/liba/libassuan/libassuan_2.5.2.orig.tar.bz2.asc;name=libassuan_2.5.2.orig.tar.bz2.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/liba/libassuan/libassuan_2.5.2-1.debian.tar.xz;name=libassuan_2.5.2-1.debian.tar.xz \
 "
 
 SRC_URI[libassuan_2.5.2-1.dsc.md5sum] = "b698bc02f2e9c5a6228b8876106909c8"
diff --git a/meta-rz-common/recipes-debian/buster/sources/libgcrypt20.inc b/meta-rz-common/recipes-debian/buster/sources/libgcrypt20.inc
index 9cc52434..826b5789 100644
--- a/meta-rz-common/recipes-debian/buster/sources/libgcrypt20.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/libgcrypt20.inc
@@ -5,10 +5,10 @@ REPACK_PV = "1.8.4"
 PV = "1.8.4"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4-5+deb10u1.dsc;name=libgcrypt20_1.8.4-5+deb10u1.dsc \
-    ${DEBIAN_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4.orig.tar.bz2;name=libgcrypt20_1.8.4.orig.tar.bz2 \
-    ${DEBIAN_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4.orig.tar.bz2.asc;name=libgcrypt20_1.8.4.orig.tar.bz2.asc \
-    ${DEBIAN_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4-5+deb10u1.debian.tar.xz;name=libgcrypt20_1.8.4-5+deb10u1.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4-5+deb10u1.dsc;name=libgcrypt20_1.8.4-5+deb10u1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4.orig.tar.bz2;name=libgcrypt20_1.8.4.orig.tar.bz2 \
+    ${DEBIAN_ELTS_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4.orig.tar.bz2.asc;name=libgcrypt20_1.8.4.orig.tar.bz2.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/libg/libgcrypt20/libgcrypt20_1.8.4-5+deb10u1.debian.tar.xz;name=libgcrypt20_1.8.4-5+deb10u1.debian.tar.xz \
 "
 
 SRC_URI[libgcrypt20_1.8.4-5+deb10u1.dsc.md5sum] = "815ccee6590f9806ea1ceaad608e4bb3"
diff --git a/meta-rz-common/recipes-debian/buster/sources/libunistring.inc b/meta-rz-common/recipes-debian/buster/sources/libunistring.inc
index c9f4528a..9c03a36f 100644
--- a/meta-rz-common/recipes-debian/buster/sources/libunistring.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/libunistring.inc
@@ -5,10 +5,10 @@ REPACK_PV = "0.9.10"
 PV = "0.9.10"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/libu/libunistring/libunistring_0.9.10-1.dsc;name=libunistring_0.9.10-1.dsc \
-    ${DEBIAN_MIRROR}/main/libu/libunistring/libunistring_0.9.10.orig.tar.xz;name=libunistring_0.9.10.orig.tar.xz \
-    ${DEBIAN_MIRROR}/main/libu/libunistring/libunistring_0.9.10.orig.tar.xz.asc;name=libunistring_0.9.10.orig.tar.xz.asc \
-    ${DEBIAN_MIRROR}/main/libu/libunistring/libunistring_0.9.10-1.debian.tar.xz;name=libunistring_0.9.10-1.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/libu/libunistring/libunistring_0.9.10-1.dsc;name=libunistring_0.9.10-1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/libu/libunistring/libunistring_0.9.10.orig.tar.xz;name=libunistring_0.9.10.orig.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/libu/libunistring/libunistring_0.9.10.orig.tar.xz.asc;name=libunistring_0.9.10.orig.tar.xz.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/libu/libunistring/libunistring_0.9.10-1.debian.tar.xz;name=libunistring_0.9.10-1.debian.tar.xz \
 "
 
 SRC_URI[libunistring_0.9.10-1.dsc.md5sum] = "cc121928372051a9b24fa3e4c4588214"
diff --git a/meta-rz-common/recipes-debian/buster/sources/nss.inc b/meta-rz-common/recipes-debian/buster/sources/nss.inc
index 62063a41..73f1b951 100644
--- a/meta-rz-common/recipes-debian/buster/sources/nss.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/nss.inc
@@ -5,9 +5,9 @@ REPACK_PV = "3.42.1"
 PV = "3.42.1"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/n/nss/nss_3.42.1-1+deb10u3.dsc;name=nss_3.42.1-1+deb10u3.dsc \
-    ${DEBIAN_MIRROR}/main/n/nss/nss_3.42.1.orig.tar.gz;name=nss_3.42.1.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/n/nss/nss_3.42.1-1+deb10u3.debian.tar.xz;name=nss_3.42.1-1+deb10u3.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/n/nss/nss_3.42.1-1+deb10u3.dsc;name=nss_3.42.1-1+deb10u3.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/n/nss/nss_3.42.1.orig.tar.gz;name=nss_3.42.1.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/n/nss/nss_3.42.1-1+deb10u3.debian.tar.xz;name=nss_3.42.1-1+deb10u3.debian.tar.xz \
 "
 
 SRC_URI[nss_3.42.1-1+deb10u3.dsc.md5sum] = "171dc93a591104f6180adcf112e20d39"
diff --git a/meta-rz-common/recipes-debian/buster/sources/openssh.inc b/meta-rz-common/recipes-debian/buster/sources/openssh.inc
index 2951afc9..d594a9b2 100644
--- a/meta-rz-common/recipes-debian/buster/sources/openssh.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/openssh.inc
@@ -1,22 +1,22 @@
 # This is generated by debian-source.bbclass
-DPV = "7.9p1-10+deb10u2"
+DPV = "7.9p1-10+deb10u5"
 DPV_EPOCH = "1"
 REPACK_PV = "7.9p1"
 PV = "7.9p1"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/o/openssh/openssh_7.9p1-10+deb10u2.dsc;name=openssh_7.9p1-10+deb10u2.dsc \
-    ${DEBIAN_MIRROR}/main/o/openssh/openssh_7.9p1.orig.tar.gz;name=openssh_7.9p1.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/o/openssh/openssh_7.9p1.orig.tar.gz.asc;name=openssh_7.9p1.orig.tar.gz.asc \
-    ${DEBIAN_MIRROR}/main/o/openssh/openssh_7.9p1-10+deb10u2.debian.tar.xz;name=openssh_7.9p1-10+deb10u2.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssh/openssh_7.9p1-10+deb10u5.dsc;name=openssh_7.9p1-10+deb10u5.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssh/openssh_7.9p1.orig.tar.gz;name=openssh_7.9p1.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssh/openssh_7.9p1.orig.tar.gz.asc;name=openssh_7.9p1.orig.tar.gz.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssh/openssh_7.9p1-10+deb10u5.debian.tar.xz;name=openssh_7.9p1-10+deb10u5.debian.tar.xz \
 "
 
-SRC_URI[openssh_7.9p1-10+deb10u2.dsc.md5sum] = "4687cc10eb11fb78972061a2e5b4c219"
+SRC_URI[openssh_7.9p1-10+deb10u5.dsc.md5sum] = "713faef2f72d0a83a0ac2eab0e9a17ba"
 SRC_URI[openssh_7.9p1.orig.tar.gz.md5sum] = "c6af50b7a474d04726a5aa747a5dce8f"
 SRC_URI[openssh_7.9p1.orig.tar.gz.asc.md5sum] = "36f798ff56b24004e16c08bf4a5fc021"
-SRC_URI[openssh_7.9p1-10+deb10u2.debian.tar.xz.md5sum] = "e235664e9191eb5a4b91f30c03c9de0a"
+SRC_URI[openssh_7.9p1-10+deb10u5.debian.tar.xz.md5sum] = "bb3487b5799f190c33371a4a43614466"
 
-SRC_URI[openssh_7.9p1-10+deb10u2.dsc.sha256sum] = "bb8384534491eb1edba6c12a2d4f289e59abb4ec4795101d9655fae52e426dec"
+SRC_URI[openssh_7.9p1-10+deb10u5.dsc.sha256sum] = "63f4d3bb0bc4113f8fb43585da21d9629d9ec870ba2937cc43c1297bf887053c"
 SRC_URI[openssh_7.9p1.orig.tar.gz.sha256sum] = "6b4b3ba2253d84ed3771c8050728d597c91cfce898713beb7b64a305b6f11aad"
 SRC_URI[openssh_7.9p1.orig.tar.gz.asc.sha256sum] = "4fd584498595450d68f5514b3d79eb14425a3d6aa9e9021d9e928fdd7b4469eb"
-SRC_URI[openssh_7.9p1-10+deb10u2.debian.tar.xz.sha256sum] = "2f8d81757e3050aab4ab735692c1b2d521cee54ec3987ef3b08fbaff84abf3c1"
+SRC_URI[openssh_7.9p1-10+deb10u5.debian.tar.xz.sha256sum] = "211a2937f6e9c9d2c19594a1a2887cb285910f1ddaacbc441888f69add01cb80"
diff --git a/meta-rz-common/recipes-debian/buster/sources/openssl.inc b/meta-rz-common/recipes-debian/buster/sources/openssl.inc
index 8588a81c..731ba196 100644
--- a/meta-rz-common/recipes-debian/buster/sources/openssl.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/openssl.inc
@@ -1,22 +1,22 @@
 # This is generated by debian-source.bbclass
-DPV = "1.1.1n-0+deb10u6"
+DPV = "1.1.1n-0+deb10u7"
 DPV_EPOCH = ""
 REPACK_PV = "1.1.1n"
 PV = "1.1.1n"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/o/openssl/openssl_1.1.1n-0+deb10u6.dsc;name=openssl_1.1.1n-0+deb10u6.dsc \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/o/openssl/openssl_1.1.1n.orig.tar.gz;name=openssl_1.1.1n.orig.tar.gz \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/o/openssl/openssl_1.1.1n.orig.tar.gz.asc;name=openssl_1.1.1n.orig.tar.gz.asc \
-    ${DEBIAN_SECURITY_UPDATE_MIRROR}/main/o/openssl/openssl_1.1.1n-0+deb10u6.debian.tar.xz;name=openssl_1.1.1n-0+deb10u6.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssl/openssl_1.1.1n-0+deb10u7.dsc;name=openssl_1.1.1n-0+deb10u7.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssl/openssl_1.1.1n.orig.tar.gz;name=openssl_1.1.1n.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssl/openssl_1.1.1n.orig.tar.gz.asc;name=openssl_1.1.1n.orig.tar.gz.asc \
+    ${DEBIAN_ELTS_MIRROR}/main/o/openssl/openssl_1.1.1n-0+deb10u7.debian.tar.xz;name=openssl_1.1.1n-0+deb10u7.debian.tar.xz \
 "
 
-SRC_URI[openssl_1.1.1n-0+deb10u6.dsc.md5sum] = "5436865478157ca5853c240c42b0393c"
+SRC_URI[openssl_1.1.1n-0+deb10u7.dsc.md5sum] = "69e79f75ac983e13727d49f97fafe0c1"
 SRC_URI[openssl_1.1.1n.orig.tar.gz.md5sum] = "2aad5635f9bb338bc2c6b7d19cbc9676"
 SRC_URI[openssl_1.1.1n.orig.tar.gz.asc.md5sum] = "55cfa25c4358a6e2f8bb6ad7dd0b9f93"
-SRC_URI[openssl_1.1.1n-0+deb10u6.debian.tar.xz.md5sum] = "bf6f07f3efd9360faef5fcfb618cb89f"
+SRC_URI[openssl_1.1.1n-0+deb10u7.debian.tar.xz.md5sum] = "bb46671fa8c7ab16285ab22c33eb8c94"
 
-SRC_URI[openssl_1.1.1n-0+deb10u6.dsc.sha256sum] = "7581c6b6f2b194f23673da136c2e2697c0ca69462fb9daa0e344504ae12042e8"
+SRC_URI[openssl_1.1.1n-0+deb10u7.dsc.sha256sum] = "859720009d50fe99ec9a52488796b4ba54a6d765fc4bef5f2cd1f6bcac68256f"
 SRC_URI[openssl_1.1.1n.orig.tar.gz.sha256sum] = "40dceb51a4f6a5275bde0e6bf20ef4b91bfc32ed57c0552e2e8e15463372b17a"
 SRC_URI[openssl_1.1.1n.orig.tar.gz.asc.sha256sum] = "e0e89e9467102880ee6f2ee8c1413933eb1268969afb97b9bec61e2190a62fd0"
-SRC_URI[openssl_1.1.1n-0+deb10u6.debian.tar.xz.sha256sum] = "95e5d25cc39d3b50ffa4e1c83477540d0b34726dee2b812372e005e6ce44f6fc"
+SRC_URI[openssl_1.1.1n-0+deb10u7.debian.tar.xz.sha256sum] = "041a5e9e57fc3842c4b990cdbc641c989daf81a72753ed001ef1e5dfd41e8e7e"
diff --git a/meta-rz-common/recipes-debian/buster/sources/perl.inc b/meta-rz-common/recipes-debian/buster/sources/perl.inc
index 2e7b61e6..f281b1e5 100644
--- a/meta-rz-common/recipes-debian/buster/sources/perl.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/perl.inc
@@ -5,10 +5,10 @@ REPACK_PV = "5.28.1"
 PV = "5.28.1"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/p/perl/perl_5.28.1-6+deb10u1.dsc;name=perl_5.28.1-6+deb10u1.dsc \
-    ${DEBIAN_MIRROR}/main/p/perl/perl_5.28.1.orig-regen-configure.tar.xz;name=perl_5.28.1.orig-regen-configure.tar.xz \
-    ${DEBIAN_MIRROR}/main/p/perl/perl_5.28.1.orig.tar.xz;name=perl_5.28.1.orig.tar.xz \
-    ${DEBIAN_MIRROR}/main/p/perl/perl_5.28.1-6+deb10u1.debian.tar.xz;name=perl_5.28.1-6+deb10u1.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/p/perl/perl_5.28.1-6+deb10u1.dsc;name=perl_5.28.1-6+deb10u1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/p/perl/perl_5.28.1.orig-regen-configure.tar.xz;name=perl_5.28.1.orig-regen-configure.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/p/perl/perl_5.28.1.orig.tar.xz;name=perl_5.28.1.orig.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/p/perl/perl_5.28.1-6+deb10u1.debian.tar.xz;name=perl_5.28.1-6+deb10u1.debian.tar.xz \
 "
 
 SRC_URI[perl_5.28.1-6+deb10u1.dsc.md5sum] = "f075eeb566d92f3a03eba415889991d4"
diff --git a/meta-rz-common/recipes-debian/buster/sources/pkg-config.inc b/meta-rz-common/recipes-debian/buster/sources/pkg-config.inc
index 68369c46..25fe45c3 100644
--- a/meta-rz-common/recipes-debian/buster/sources/pkg-config.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/pkg-config.inc
@@ -5,9 +5,9 @@ REPACK_PV = "0.29"
 PV = "0.29"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/p/pkg-config/pkg-config_0.29-6.dsc;name=pkg-config_0.29-6.dsc \
-    ${DEBIAN_MIRROR}/main/p/pkg-config/pkg-config_0.29.orig.tar.gz;name=pkg-config_0.29.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/p/pkg-config/pkg-config_0.29-6.diff.gz;name=pkg-config_0.29-6.diff.gz;apply=no \
+    ${DEBIAN_ELTS_MIRROR}/main/p/pkg-config/pkg-config_0.29-6.dsc;name=pkg-config_0.29-6.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/p/pkg-config/pkg-config_0.29.orig.tar.gz;name=pkg-config_0.29.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/p/pkg-config/pkg-config_0.29-6.diff.gz;name=pkg-config_0.29-6.diff.gz;apply=no \
 "
 
 SRC_URI[pkg-config_0.29-6.dsc.md5sum] = "f40a3ea1c3cb1b7c514213395740634c"
diff --git a/meta-rz-common/recipes-debian/buster/sources/pseudo.inc b/meta-rz-common/recipes-debian/buster/sources/pseudo.inc
index 4564fd1c..41c9c247 100644
--- a/meta-rz-common/recipes-debian/buster/sources/pseudo.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/pseudo.inc
@@ -5,9 +5,9 @@ REPACK_PV = "1.9.0+git20180920"
 PV = "1.9.0+git20180920"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/p/pseudo/pseudo_1.9.0+git20180920-1.dsc;name=pseudo_1.9.0+git20180920-1.dsc \
-    ${DEBIAN_MIRROR}/main/p/pseudo/pseudo_1.9.0+git20180920.orig.tar.gz;name=pseudo_1.9.0+git20180920.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/p/pseudo/pseudo_1.9.0+git20180920-1.debian.tar.xz;name=pseudo_1.9.0+git20180920-1.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/p/pseudo/pseudo_1.9.0+git20180920-1.dsc;name=pseudo_1.9.0+git20180920-1.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/p/pseudo/pseudo_1.9.0+git20180920.orig.tar.gz;name=pseudo_1.9.0+git20180920.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/p/pseudo/pseudo_1.9.0+git20180920-1.debian.tar.xz;name=pseudo_1.9.0+git20180920-1.debian.tar.xz \
 "
 
 SRC_URI[pseudo_1.9.0+git20180920-1.dsc.md5sum] = "1a224e4fe7b891dde34b96a115d4ff9f"
diff --git a/meta-rz-common/recipes-debian/buster/sources/quilt.inc b/meta-rz-common/recipes-debian/buster/sources/quilt.inc
index 8f609ce3..8ac9ad3b 100644
--- a/meta-rz-common/recipes-debian/buster/sources/quilt.inc
+++ b/meta-rz-common/recipes-debian/buster/sources/quilt.inc
@@ -5,9 +5,9 @@ REPACK_PV = "0.65"
 PV = "0.65"
 
 DEBIAN_SRC_URI = " \
-    ${DEBIAN_MIRROR}/main/q/quilt/quilt_0.65-3.dsc;name=quilt_0.65-3.dsc \
-    ${DEBIAN_MIRROR}/main/q/quilt/quilt_0.65.orig.tar.gz;name=quilt_0.65.orig.tar.gz \
-    ${DEBIAN_MIRROR}/main/q/quilt/quilt_0.65-3.debian.tar.xz;name=quilt_0.65-3.debian.tar.xz \
+    ${DEBIAN_ELTS_MIRROR}/main/q/quilt/quilt_0.65-3.dsc;name=quilt_0.65-3.dsc \
+    ${DEBIAN_ELTS_MIRROR}/main/q/quilt/quilt_0.65.orig.tar.gz;name=quilt_0.65.orig.tar.gz \
+    ${DEBIAN_ELTS_MIRROR}/main/q/quilt/quilt_0.65-3.debian.tar.xz;name=quilt_0.65-3.debian.tar.xz \
 "
 
 SRC_URI[quilt_0.65-3.dsc.md5sum] = "599df8c00bbb54edbf2a365f6d39271c"
-- 
2.17.1

